practice_problems_2/removesOdd/main.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$removes_odd = function (array $input) {
+    $counter = 0;
+
+    while (count($input) > $counter) {
+        if ($input[$counter] % 2 != 0) {
+            unset($input[$counter]);
+            $input = array_values($input);  
+    } 
+    else {
+    $counter++;
```

Comment:
Code indentation issue here. Use linter in vs code

Reply:
Fixed